atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -47,15 +49,13 @@ public File acceptorLogDir() {
     public abstract String localServer();
 
     @Size(min = 1)
-    public abstract Set<String> leaders();
+    public abstract List<String> leaders();
 
     @JsonProperty("lockCreator")
     @Value.Default
     public String lockCreator() {
-        if (leaders().isEmpty()) {
-            throw new IllegalStateException("The leaders block cannot be empty");
-        }
-        return leaders().iterator().next();
+        Preconditions.checkState(!leaders().isEmpty(), "The leaders block cannot be empty");
+        return Iterables.getFirst(leaders(), null);
```

Comment:
You're right. `Iterables.getLast(x)` exists, so I assumed there was an analogue. 

Sorry, guess I wasn't clear.  My point was that `Set.iterator().next()` (before) might be different from `List.iterator().next()` (this PR).  Though like I said, I don't think this is likely to be an issue.